<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>670</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dinar Kendin Yap Atöyeleri</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="uploadButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>210</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Formu Yükle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="drawButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>80</y>
     <width>210</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kura Çek</string>
   </property>
  </widget>
  <widget class="QLabel" name="picture2">
   <property name="geometry">
    <rect>
     <x>1039</x>
     <y>19</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>ilce_mem_logo.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="picture1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>29</y>
     <width>181</width>
     <height>181</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap>kendin_yap_logo.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="header">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Dinar Kendin Yap Atölyeleri Kura Programı</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gifLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>1200</width>
     <height>411</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textSinif3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3. Sınıflar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="textSinif4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4. Sınıflar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="textSinif5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>230</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5. Sınıflar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="textSinif6">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>230</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6. Sınıflar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="textSinif7">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>230</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7. Sınıflar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="liste3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>230</width>
     <height>360</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="liste4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>230</width>
     <height>360</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="liste5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>270</y>
     <width>230</width>
     <height>360</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="liste6">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>270</y>
     <width>230</width>
     <height>360</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="liste7">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>270</y>
     <width>230</width>
     <height>360</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>640</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>github.com/NurettinSelim</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
